modules/Kconfig.mcuboot

```diff
@@ -77,3 +77,9 @@ config NRF53_RAM_FLASH_SIZE
 	  is read from external flash, and buffered in  RAM. This option is used
 	  to define forward the size of this RAM region to the partition manager
 	  configuration.
+
+config MCUBOOT_TFM_PRESENT
+	bool "TF-M is present"
```

Comment:
Should this depend on MCUBOOT? So that it does not pollute other images than MCUboot itself

Reply:
You are right, I thought that Kconfig.mcuboot will have that restriction by default but I was wrong. I added this.